source/agora/utils/Log.d

```diff
@@ -111,16 +111,25 @@ public alias Log = Ocean.Log;
 static this ()
 {
     version (unittest)
-        auto appender = CircularAppender();
+        auto appender = getCircularAppender();
     else
         auto appender = new AppendConsole();
 
     appender.layout(new AgoraLayout());
     Log.root.add(appender);
 }
 
+public CircularAppender!BufferSize getCircularAppender (size_t BufferSize = 2^^20)()
+{
+    static CircularAppender!(BufferSize) appender;
+        if (appender is null)
+            appender = new CircularAppender!(BufferSize)();
+
+    return appender;
+}
+
```

Comment:
Why the `opCall` removal ? I understand you had to template the class for testing purpose, and it makes sense, but you could just put the default value on the `class` and change the calls from `CircularBuffer()` to `CircularBuffer!()()`.

Reply:
I think I just didn't find CircularBuffer!()() pretty, but I changed it now

Reply:
Yeh it's not. Renaming `opCall` to `get` would also work, but it should be a separate commit at least.